### ISSUE TITLE
[loc] Request to add English (United Kingdom) language to VS Code

### ISSUE THREAD

**Issue by mjcheetham** [Microsoft/vscode#35690]
I have submitted requests for the new language on all vscode-* transifex projects.
I am willing to translate to English (United Kingdom).

My transifex account name is "mjcheetham".

**Comment by talamaska** [comment]
what's different from the current default English?

**Comment by mjcheetham** [comment]
There are a number of difference spellings (and differences between nouns or verbs in spelling) for some words. A few examples:

#### Color :us: vs Colour :gb:
![image](https://user-images.githubusercontent.com/5658207/31278386-ab0d4108-aa9c-11e7-979c-1097437f15ae.png)
![image](https://user-images.githubusercontent.com/5658207/31278404-bf09f9ee-aa9c-11e7-8277-8cb1a42d1fac.png)
![image](https://user-images.githubusercontent.com/5658207/31278406-c178aac2-aa9c-11e7-8bcd-399ab0cd11a1.png)

#### Center :us: vs Centre :gb:
![image](https://user-images.githubusercontent.com/5658207/31278544-38c51ade-aa9d-11e7-9d46-f74c242b19fa.png)


**NB:** I'm not suggesting changing the CSS properties here :sweat_smile:, just the help text and descriptions. Admittedly some of these strings here might be from the CSS language service and not part of the editor or workbench directly, but they are bundled with Code.



#### -ize :us: vs -ise :gb:
![image](https://user-images.githubusercontent.com/5658207/31278424-d40140f0-aa9c-11e7-8e3b-e05aea97f22a.png)
![image](https://user-images.githubusercontent.com/5658207/31278432-db8d0a48-aa9c-11e7-835b-2249dc0c4e48.png)
![image](https://user-images.githubusercontent.com/5658207/31278434-dd42b75c-aa9c-11e7-9cef-a8b556caaa0b.png)

#### License (n., v.) :us: vs Licence (n.) and License (v.) :gb:
![image](https://user-images.githubusercontent.com/5658207/31278487-03ea8db2-aa9d-11e7-83ad-dc420fd5bf0f.png)

**Comment by danyeh** [comment]
@mjcheetham , please review Adding language criteria here: https://github.com/Microsoft/Localization/wiki/How-to-request-a-new-UI-language-for-VS-Code-project.
And add your Transifex account link as a vote in this tracking issue.

**Comment by mjcheetham** [comment]
Transifex account is:
mjcheetham

_(https://www.transifex.com/user/profile/mjcheetham/)_

**Comment by mjcheetham** [comment]
I've noticed that in the past few months the English (United Kingdom) language has appeared on Transifex as a language available to translation, with currently four members listed as translators for the [UK team](https://www.transifex.com/microsoft-oss/teams/72408/en_GB/).

- BenjaminSouchet
- LogneBudo
- lasithds
- mjcheetham (myself)

I've made many suggested translations to various strings, but in the last few days there seems to have been many strings marked as 'translated' (but still unreviewed) with incorrect translations (spelling mistakes from the British English perspective; for example "color" rather than "colour").

I do not seem to be able to do anything to remedy these incorrect translations, except make new suggestions or vote others (many of for which I have already made the correct suggestion).

Has UK English been formally accepted or not?

Example string which has been mistranslated as of 3 days ago:

![image](https://user-images.githubusercontent.com/5658207/40277753-64fb6e92-5c1c-11e8-8013-4867a62e1f4c.png)

![image](https://user-images.githubusercontent.com/5658207/40277755-6de116e2-5c1c-11e8-86aa-889b3ef4dabc.png)
